Bonanza_LTER

juliebielecki opened the pull request:
linking to issue #42

juliebielecki committed:
intital template checklist, all but citation completed

juliebielecki committed:
submission ready for external review, updated references and bib too

ktoddbrown commented on data_reports/ISCN3_bibliography.bib:
```diff
@@ -144,4 +144,27 @@ @misc{Zinke1986
+  doi = {10.1139/x93-122},
+  url = {https://doi.org/10.1139/x93-122},
+  year = {1993},
+  month = may,
```
Add {} around the month to make it consistent with the rest of the entry.

ktoddbrown committed:
Update ISCN3_bibliography.bib

Added "" or {} around month. Possible that this is the wrong move but it makes the entry more consistant.

ktoddbrown reviewed (approved):
Looks good!
